adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/adidas_ascii_logoTests.swift

```diff
@@ -0,0 +1,111 @@
+        @@@@@@@   @@@@@@@   @@@@@@@
+         @@@@@@@   @@@@@@@   @@@@@@@
+          @@@@@@@   @@@@@@@   @@@@@@@
+        """
+        XCTAssertEqual(createAdidasAsciiLogo(width:7),res)
+    }
+    
+    func testWidth9() {
+        let res = """
+                          @@@@@@@@@
+                           @@@@@@@@@
+                            @@@@@@@@@
+                 @@@@@@@@@   @@@@@@@@@
+                  @@@@@@@@@   @@@@@@@@@
+                   @@@@@@@@@   @@@@@@@@@
+        @@@@@@@@@   @@@@@@@@@   @@@@@@@@@
+         @@@@@@@@@   @@@@@@@@@   @@@@@@@@@
+          @@@@@@@@@   @@@@@@@@@   @@@@@@@@@
+        """
+        XCTAssertEqual(createAdidasAsciiLogo(width:9),res)
+    }
+    func testWidth16() {
+        let res = """
+                                         @@@@@@@@@@@@@@@@
+                                         @@@@@@@@@@@@@@@@
+                                          @@@@@@@@@@@@@@@@
+                                           @@@@@@@@@@@@@@@@
+                        @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+                         @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+                          @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+                           @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+        @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+         @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+          @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+           @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@    @@@@@@@@@@@@@@@@
+        """
+        XCTAssertEqual(createAdidasAsciiLogo(width:16),res)
+    }
+    
+    func testWidth21() {
+        let res = """
+                                                  @@@@@@@@@@@@@@@@@@@@@
+                                                   @@@@@@@@@@@@@@@@@@@@@
+                                                    @@@@@@@@@@@@@@@@@@@@@
+                                                     @@@@@@@@@@@@@@@@@@@@@
+                                                      @@@@@@@@@@@@@@@@@@@@@
+                             @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+                              @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+                               @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+                                @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+                                 @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+        @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+         @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+          @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+           @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+            @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+        """
+        XCTAssertEqual(createAdidasAsciiLogo(width:21),res)
+    }
+    
+
+    static var allTests = [
+        ("testWidth2", testWidth2),
```

Comment:
Should we add here all the other test cases?

Reply:
Done